### PR TITLE
build: Cupcake

### PR DESCRIPTION
**Pull request by vorburger**

### CODE REVIEW

File: .github/workflows/mega-linter.yaml [hunk 1]
```diff
@@ -72,9 +72,7 @@ jobs:
       # MegaLinter
       - name: MegaLinter
 
-        # You can override MegaLinter flavor used to have faster performances
-        # More info at https://megalinter.io/flavors/
-        uses: oxsecurity/megalinter@b48455a119cc28045eee8f1e9d0a542a85e71f4f # v7.5.0
+        uses: oxsecurity/megalinter/flavors/cupcake@v7.5.0
```
Comment (vorburger) [inline .github/workflows/mega-linter.yaml:75]: How does one find the SHA? I can see https://ghcr.io/oxsecurity/megalinter but there's nothing at https://ghcr.io/oxsecurity/megalinter/flavors/cupcake ... @echoix would you happen to know this? (I could contrib an improvement to your docs.)

Comment (echoix) [reply .github/workflows/mega-linter.yaml:75]: Here, in GitHub workflows, the `uses:` refers to an action, not a docker image. Not all Actions use docker images. So I expect that the sha you are looking for is the commit sha of the tagged release. The flavor is an action file that is in a subdirectory of the repo (as opposed to the one in the root of the repo)

Comment (vorburger) [reply .github/workflows/mega-linter.yaml]: Oh, je vois! Qu'est ce que j'suis bête... là j'ai compris. Merci bcp, j'apprécie la réponse/clarification - TIL!

Comment (echoix) [reply .github/workflows/mega-linter.yaml]: Pas de problème!

**Pull request merged**